sphinx-doc/howto/molecular.py

```diff
@@ -1,27 +1,24 @@
-# Copyright (c) 2009-2023 The Regents of the University of Michigan.
-# Part of HOOMD-blue, released under the BSD 3-Clause License.
-
 import hoomd
 import gsd.hoomd
 
-snapshot = gsd.hoomd.Snapshot()
+frame = gsd.hoomd.Frame()
```

Comment:
Should changes to be compatible with the upcoming GSD 3.0 release be on a different PR and/or delayed until after GSD 3.0 is released?

Reply:
`Frame` is available now in GSD 2.8.0.